src/LAMMPS.jl

```diff
@@ -233,6 +241,48 @@ function command(lmp::LMP, cmd::Union{String, Array{String}})
     check(lmp)
 end
 
+"""
+    create_atoms(
+        lmp::LMP, x::Matrix{Float64}, id::Vector{Int32}, types::Vector{Int32};
+        v::Union{Ptr{Float64},Matrix{Float64}}=Ptr{Float64}(C_NULL),
+        image::Union{Ptr{IMAGEINT},Vector{IMAGEINT}}=Ptr{IMAGEINT}(C_NULL),
+        bexpand::Bool=false
+    )
+
+Create atoms for a LAMMPS instance. 
+x contains the atom positions and should be a 3 by Matrix{Float64}, where n is the number of atoms. 
+id contains the id of each atom and should all be Vector{Int32} with length n.
+types contains the atomic type (LAMMPS number) of each atom and should all be a Vector{Int32} with length n.
+v contains the associated velocities and should be a 3 by Matrix{Float64}.
+image contains the image flags for each atom and should be Vector{IMAGEINT} with length n.
+bexpand is a Bool that defines whether or not the box should be expanded to fit the input atoms (default not).
+"""
+function create_atoms(
+    lmp::LMP, x::Matrix{Float64}, id::Vector{Int32}, types::Vector{Int32};
+    v::Union{Ptr{Float64},Matrix{Float64}}=Ptr{Float64}(C_NULL),
```

Comment:
This `v` being a raw pointer makes me nervous. It incentivies the user to write `pointer(v)` which is wrong in most circumstances. Same for image. 

To denote that this can be null, use `Nothing`